README.md

```diff
@@ -1,2 +1,133 @@
 # API-manifesto
 Documents how to write APIs
+
+## Response
+
+Generally we have a few rules the response has to follow:
+
+When we don’t have any data, we need to return in the following way:
+Consistency of key types. e.g. always return IDs as an integer in all endpoints.
+Date/timestamps should always be returned with a time zone.
+Pagination data should be returned in a meta key.
+When an endpoint doesn't have meaningful data to return (e.g. when deleting something), use a status key to communicate the status of the endpoint.
+
+### Body
+
+#### Object at the root level
+
+A body should always return an object at the root level. This ensures flexibiliy if the response needs to change in the future. We recommend using `data` for succesful requests with meaningful response data and `error` for unsuccesful requests with error data being returned.
+
+##### Do
```

Comment:
Instead of these headers we could consider:
✅  ✔️  ➕  👍 
⛔️  ❌  🔴  ❗️  ‼️  ➖  👎